docs/standard/io/index.md

```diff
@@ -22,7 +22,7 @@ ms.author: "mairaw"
 manager: "wpickett"
 ---
 # File and Stream I/O
-File and stream I/O (input/output) refers to the transfer of data either to or from a storage medium. In the .NET Framework, the [System.IO](http://go.microsoft.com/fwlink/?LinkId=231142) namespaces contain types that enable reading and writing, both synchronously and asynchronously, on data streams and files. These namespaces also contain types that perform compression and decompression on files, and types that enable communication through pipes and serial ports.  
+File and stream I/O (input/output) refers to the transfer of data either to or from a storage medium. In the .NET Framework, the [System.IO](http://msdn.microsoft.com/en-us/library/gg145019(v=VS.110).aspx) namespaces contain types that enable reading and writing, both synchronously and asynchronously, on data streams and files. These namespaces also contain types that perform compression and decompression on files, and types that enable communication through pipes and serial ports.  
```

Comment:
Change this link to https://docs.microsoft.com/dotnet/api/?term=system.io&view=netframework-4.7.1